gatsby-source-graphcms/src/gatsby-node.js

```diff
@@ -334,37 +341,85 @@ const generateImageSource = (
   return { src, width, height, format }
 }
 
-const resolveGatsbyImageData = async (
-  { handle: filename, height, mimeType, width },
-  options
-) => {
-  const imageDataArgs = {
-    ...options,
-    pluginName: `gatsby-source-graphcms`,
-    sourceMetadata: { format: mimeType.split('/')[1], height, width },
-    filename,
-    generateImageSource,
-    options,
-  }
+function makeResolveGatsbyImageData(cache) {
+  return async function resolveGatsbyImageData(
+    { handle: filename, height, mimeType, width, url, internal },
+    options
+  ) {
+    const { placeholder = 'BLURRED' } = options
+
+    const imageDataArgs = {
+      ...options,
+      pluginName: `gatsby-source-graphcms`,
```

Comment:
Perhaps we can import `package.json` above, and use the `name` from there as that constant is already set.

```js
import pkg from '../package.json`
```

Reply:
I ran into issues while importing the JSON file, so I created a `constants` file on the util folder, and where I found the plugin name, I've refactored to use this constant. Let me know if this is okay.